### ISSUE TITLE
Missing request header to make amazon.com bookmarks work

### ISSUE THREAD

**Issue by felixdv** [xwmx/nb#246]
I noticed that fetching the content for bookmarks of amazon.com products seemed to consistently go wrong with a 503 HTTP error and the message "Sorry! Something went wrong!".

In my case, `nb` is using `curl` and while it does have the user agent set with `$NB_USER_AGENT`, it's missing an `Accept-Encoding` header. Turns out that without this header, Amazon throws a 503 error.

Quick test case with curl:

```bash
$ curl -s -o /dev/null -w "%{http_code}" -A "nb/7.4.1" -H "Accept-Encoding: *" \
    https://www.amazon.com/Linux-Kernel-Development-Robert-Love/dp/0672329468
200%

$ curl -s -o /dev/null -w "%{http_code}" -A "nb/7.4.1" \
    https://www.amazon.com/Linux-Kernel-Development-Robert-Love/dp/0672329468
503%
```

Adding an `Accept-Encoding: *` header to curl/wget would fix the issue. If this solution is acceptable, I can make a PR for this.

**Comment by xwmx** [comment]
@felixdv Sounds good. Thanks!

**Comment by xwmx** [comment]
I went ahead and added this header to `curl` and `wget` in `download_from()`, and it appears to be working with amazon.com bookmarks now. This is available in the latest version in the repo and will be included in the next release version. Let me know if you run into any issues with it.

**Comment by xwmx** [comment]
This is available as of version 7.5.0. @felixdv Thanks for the heads up!
